src/Crocodealer/Config.hs

```diff
@@ -0,0 +1,17 @@
+module Crocodealer.Config
+       ( Repo(..)
+       , Config(..)
+       ) where
+
+import Crocodealer.Core.Label
+
```

Comment:
There should be two empty lines after the import section according to our style guide
```suggestion


```